my_type=hypothesis
proj=hdfs
u_test=org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
h_list[1]=ipc.server.max.connections@@@hdfs:NameNode@@@-1@@@0@@@1
proj: hdfs
u_test: org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
h_list: ipc.server.max.connections@@@hdfs:NameNode@@@-1@@@0@@@1
updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v1v2 running time = 12202198780
---> v1v2 test failed.
proj: hdfs
u_test: org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
h_list: ipc.server.max.connections@@@hdfs:NameNode@@@-1@@@0@@@1
vv_mode: v1v2
result: FAIL
failureMessage: null
stackTrace: java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA.testObserverReadProxyProviderWithDT(TestDelegationTokensWithHA.java:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)


updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v1v1 running time = 12008962173
updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v2v2 running time = 27900848886
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v1v2 running time = 13304286755
---> v1v2 test failed.
proj: hdfs
u_test: org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
h_list: ipc.server.max.connections@@@hdfs:NameNode@@@-1@@@0@@@1
vv_mode: v1v2
result: FAIL
failureMessage: null
stackTrace: java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA.testObserverReadProxyProviderWithDT(TestDelegationTokensWithHA.java:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)


updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v1v1 running time = 13076822442
updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v2v2 running time = 25409111030
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v1v2 running time = 12042350808
---> v1v2 test failed.
proj: hdfs
u_test: org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
h_list: ipc.server.max.connections@@@hdfs:NameNode@@@-1@@@0@@@1
vv_mode: v1v2
result: FAIL
failureMessage: null
stackTrace: java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA.testObserverReadProxyProviderWithDT(TestDelegationTokensWithHA.java:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)


updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v1v1 running time = 12566826270
updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v2v2 running time = 31611772274
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v1v2 running time = 14332034488
---> v1v2 test failed.
proj: hdfs
u_test: org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
h_list: ipc.server.max.connections@@@hdfs:NameNode@@@-1@@@0@@@1
vv_mode: v1v2
result: FAIL
failureMessage: null
stackTrace: java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA.testObserverReadProxyProviderWithDT(TestDelegationTokensWithHA.java:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)


updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v1v1 running time = 14657885926
updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v2v2 running time = 30945158527
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v1v2 running time = 12076310701
---> v1v2 test failed.
proj: hdfs
u_test: org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
h_list: ipc.server.max.connections@@@hdfs:NameNode@@@-1@@@0@@@1
vv_mode: v1v2
result: FAIL
failureMessage: null
stackTrace: java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA.testObserverReadProxyProviderWithDT(TestDelegationTokensWithHA.java:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)


updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v1v1 running time = 11777431466
updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v2v2 running time = 26731288064
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v1v2 running time = 14173316898
---> v1v2 test failed.
proj: hdfs
u_test: org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
h_list: ipc.server.max.connections@@@hdfs:NameNode@@@-1@@@0@@@1
vv_mode: v1v2
result: FAIL
failureMessage: null
stackTrace: java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA.testObserverReadProxyProviderWithDT(TestDelegationTokensWithHA.java:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)


updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v1v1 running time = 12946052202
updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v2v2 running time = 25387796697
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v1v2 running time = 11022577552
---> v1v2 test failed.
proj: hdfs
u_test: org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
h_list: ipc.server.max.connections@@@hdfs:NameNode@@@-1@@@0@@@1
vv_mode: v1v2
result: FAIL
failureMessage: null
stackTrace: java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA.testObserverReadProxyProviderWithDT(TestDelegationTokensWithHA.java:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)


updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v1v1 running time = 11560095672
updating test result for file org.apache.hadoop.hdfs.server.namenode.ha.TestDelegationTokensWithHA#testObserverReadProxyProviderWithDT
v2v2 running time = 32001781754
---> v1v1 and v2v2 test test suceeded.
early stop after 7 is satisfied
v1v2 failed with probability 7 out of 7
v1v1v2v2 failed with probability 0 out of 7
Total execution time in seconds : 378
0
